src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1398,6 +1698,8 @@ static uct_ib_md_ops_t uct_ib_mlx5_md_ops = {
     .dereg_multithreaded = (uct_ib_md_dereg_multithreaded_func_t)ucs_empty_function_return_unsupported,
     .mem_prefetch        = uct_ib_mlx5_mem_prefetch,
     .get_atomic_mr_id    = (uct_ib_md_get_atomic_mr_id_func_t)ucs_empty_function_return_unsupported,
+    .reg_exported_key    = uct_ib_mlx5_devx_reg_exported_key,
+    .import_exported_key = uct_ib_mlx5_devx_import_exported_key
```

Comment:
why is it defined for `uct_ib_mlx5_md_ops`? Can it be that we have `HAVE_DEVX`, but it is not supported?